test/tests/redfish10/test_redfish10_system_reset_base.py

```diff
@@ -152,8 +145,33 @@ def workflow_tasklist_status_poller(tasklist, tasktype, timeout=180):
 
     return task_errorlist
 
+
+def get_rackhd_nodetype(nodeid):
+    nodetype = ""
+    sku = ""
+    # get node info
+    mondata = fit_common.rackhdapi("/api/2.0/nodes/" + nodeid)
+    if mondata['status'] == 200:
+        # get the sku id contained in the node
+        sku = mondata['json'].get("sku")
+        if sku:
+            skudata = fit_common.rackhdapi("/api/2.0/skus/" + sku)
+            if skudata['status'] == 200:
+                nodetype = skudata['json'].get("name")
+            else:
+                if fit_common.VERBOSITY >= 2:
+                    errmsg = "Error: SKU API failed {}, return code {} ".format(sku, skudata['status'])
+                    print errmsg
+        else:
+            if fit_common.VERBOSITY >= 2:
+                errmsg = "Error: nodeid {} did not return a valid sku in get_rackhd_nodetype{}".format(nodeid, sku)
+                print errmsg
+    return nodetype
```

Comment:
Should we return something like "NOT_FOUND" or "UNKNOWN" or "??????" if we don't find the node ID or the SKU?   It seems like it would be hard to detect when something went wrong if we are just returning "".  In most cases, it will just print out "Node:  xxxxxxx" instead of "Node:  xxxxxxx Quanta-T41".    I don't know that I would detect that there is a problem there.